js/plugins/pinecone/src/index.ts

```diff
@@ -116,7 +153,11 @@ export function configurePineconeRetriever<
   params: {
     indexId: string;
     clientParams?: PineconeConfiguration;
+    /**
+     * @deprecated use contentKey instead.
```

Comment:
Yes, it's for multimodal prep. Yes, pinecone supports text and images.

Reply:
Hmm, our plugin does not support it yet, but I thought `contentKey` makes sense anyway.

I dont know about native image support in Pinecone, AFAICT we need to piggyback off of the metadata object to store / retrieve images.

Reply:
True, but we do the same thing for text. In any case, I haven't modified anything to support multimodal yet, I just noticed a missing param and gave it a future-friendly name, and changed the matching params in the called functions to match the future friendly name. (Especially since the old "textKey" was defined as "_content").

Reply:
I don't feel too strongly, the change looks harmless... only that it feels unrelated. In order to support multimodal content with pinecone we'll need to do a lot more work, potentially needing a significant redesign of the plugin.

Reply:
SGTM